examples/cpp/customserializer/OrderSerializer.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "OrderSerializer.hpp"
+#include "Order.hpp"
+
+namespace customserializer {
+
+void *OrderSerializer::fromData(const std::string &className,
+                                PdxReader &pdxReader) {
+  if (className != Order::CLASS_NAME_) {
+    return nullptr;
+  }
+
+  try {
+    auto myOrder = new Order((uint32_t)pdxReader.readLong(ORDER_ID_KEY_),
```

Comment:
Maybe a cleaner example doesn't use unsigned since the server has to store all integral types as signed.